corehq/motech/repeaters/models.py

```diff
@@ -258,8 +295,12 @@ def reset_next_attempt(self):
             self.next_attempt_at = None
             self.save()
 
+    def get_repeater_type(self):
+        name = self.__class__.__name__
```

Comment:
I think `type(self).__name__` is the recommended way to do this. There are [edge cases](https://stackoverflow.com/a/10633356/10840) where `self.__class__` and `type(self)` return different things, although they are probably rare and not much of a concern here. My preference is to use `type(self)`, first because its the standard convention, second because it avoids accessing a dunder attribute directly (avoid using names that start with underscore if possible), and third it's four less characters to type.

```suggestion
        name = type(self).__name__
```

Reply:
This is done. https://github.com/dimagi/commcare-hq/pull/30755/files#diff-dbb494984fac15fda3d29dabadf40c26c18935eaac6d96c308d52d1f7237eb8cR297